README.md

```diff
@@ -48,21 +48,21 @@ this Go package, please open an issue or pull request.
 For complete examples, please see the `examples` directory.
 
 ```go
-import "github.com/MicahParks/keyfunc"
+import keyfunc "github.com/pmishchenko-ua/jwt-keyfunc"
 ```
 
 #### A note on read-only keys
-The [`JWKS.ReadOnlyKeys`](https://pkg.go.dev/github.com/MicahParks/keyfunc#JWKS.ReadOnlyKeys) method returns a read-only
+The [`JWKS.ReadOnlyKeys`](https://pkg.go.dev/github.com/pmishchenko-ua/jwt-keyfunc#JWKS.ReadOnlyKeys) method returns a read-only
```

Comment:
update all these links to point at singlestore-labs rather than your account

Reply:
will update the links (they are in tests and examples only) in a subsequent commit in the singlestore-labs repo. The package itself will be valid anyway.